exec/src/main/asciidoc/index.adoc

```diff
@@ -46,3 +46,7 @@ include::_scan-examples.adoc[leveloffset=3]
 include::{generateddocs}/fix.adoc[leveloffset=2,tag=picocli-generated-full-manpage]
 
 include::_fix-examples.adoc[leveloffset=3]
+
+'''
+
+include::{includedir}/flatten.adoc[leveloffset=2,tag=picocli-generated-full-manpage]
```

Comment:
Please add some examples like the other commands.

Reply:
done